lib/cliCommon.js

```diff
@@ -88,11 +88,17 @@ function applyCommonOptions(program) {
     program
         .option('-h, --host [hostname]', 'specify the api host')
         .option('-nov, --no-verbose', 'supress verbose info logging', false)
+        .option(
+            '--use-old-node-sass-fork',
```

Comment:
looks like quite long option. What about adding some shortcut?
```suggestion
            '-oldn, --use-old-node-sass-fork',
```

Reply:
@bc-max do you think `oldn` is enough descriptive? 
I was thinking about a shortcut, but didn't find a good one. Open for ideas :)

Reply:
> @bc-max do you think `oldn` is enough descriptive? I was thinking about a shortcut, but didn't find a good one. Open for ideas :)

Well, I think the description will be in documentation anyway. My understanding is that user is able to use both variants `-oldn` and `--use-old-node-sass-fork`. Finally, we have many shortcuts like `-h` or `-nov` 🤔